rfcs/0015-one-signature-multihop-transfer-solution-ibc-xcm.md

```diff
@@ -150,11 +149,21 @@ Cannot express(securely) arbitrary routing and programs execution.
 
 To be handled in next RFC or implementation.
 
+## Error handling
```

Comment:
Can you please provide an example where the unwinding happens, and it succeeds? And one where it doesn't?

Reply:
sure, easy. just have read reference module. see ok. will unwind as needed if to do in Substrate same as in Go. Plus final original tracking of sent xcm packet (basically unwind for xcm in hook).